src/docs/about.md

```diff
@@ -1,11 +1,30 @@
-## NMDC Schema
+# Funding and Acknowledgements
 
-**Schema resources for the National Microbiome Data Collaborative (NMDC)**
+The work conducted by the National Microbiome Data Collaborative (https://ror.org/05cwx3318) is supported by the 
+Genomic Science Program in the U.S. Department of Energy, Office of Science, Office of Biological and Environmental 
+Research (BER) under contract numbers DE-AC02-05CH11231 (LBNL), 89233218CNA000001 (LANL), and DE-AC05-76RL01830 (PNNL).
 
-The purpose of the NMDC Schema is to define metadata for the [National Microbiome Data Collaborative (NMDC)](https://microbiomedata.org/). The [NMDC](https://microbiomedata.org/) is a multi-organizational effort to enable integrated microbiome data across diverse areas in medicine, agriculture, bioenergy, and the environment. This integrated platform facilitates comprehensive discovery of and access to multidisciplinary microbiome data in order to unlock new possibilities with microbiome data science. 
+![DOE_logo.png](images/DOE_logo.png)
 
-The NMDC schema is used during the translation process to specify how metadata elements are related.
+## Framework
 
-## Credits
+This schema was made using the [LinkML framework](https://github.com/biolink/biolinkml). LinkML is a data modeling language that is used to create data 
```

Comment:
isn't that a really old URL?

Reply:
good catch - hard to tell from the diff, but this was a link in place before I began reorg - will update.

Reply:
fixed